src/themeTemplates/react-themes/dark-theme.constants.js

```diff
@@ -0,0 +1,4 @@
+export const THEMES = {
+    LIGHT: 'light',
+    DARK: 'dark'
+}
```

Comment:
for this constant:
constants of dark & light theme, should not place in dark-theme, it's scope is above than that, better to place it globally so that react/vue/angular all can share this object, OR we can have filename - theme.constants.js to be simple